projects/npm-tools/packages/npm-scripts/contrib/prettier/prettier.js

```diff
@@ -1059,28 +1102,52 @@ function prepare(filepath) {
 		if (fs.existsSync(wrapper)) {
 			dir = scripts;
 			file = wrapper;
-		} else if (fs.existsSync(fallback)) {
+		}
+		else if (fs.existsSync(fallback)) {
 			dir = modules;
 			file = fallback;
 		}
-	} else {
-		const extension = require('vscode').extensions.getExtension(
-			'esbenp.prettier-vscode'
-		);
+	}
+	else {
+		root = getFrontendProjectsRoot(filepath);
+
+		if (root) {
+			const scripts = path.join(
+				root,
+				'projects/npm-tools/packages/npm-scripts'
+			);
+
+			const wrapper = path.join(scripts, 'src/scripts/prettier.js');
+
+			if (fs.existsSync(wrapper)) {
+				dir = scripts;
+				file = wrapper;
+			}
+		}
+		else {
+			const extension = require('vscode').extensions.getExtension(
+				'esbenp.prettier-vscode'
+			);
 
-		if (extension) {
-			dir = path.join(extension.extensionPath);
-			file = path.join(dir, 'node_modules/prettier');
+			if (extension) {
+				dir = path.join(extension.extensionPath);
+				file = path.join(dir, 'node_modules/prettier');
+			}
 		}
 	}
+
```

Comment:
This is the least satisfying part of the entire PR. I wrote it late at night, so maybe it could be cleaned up a bit, but I feel there is something inescapably ugly about all the hackery we have to do in this script.

![forgive-me](https://user-images.githubusercontent.com/7074/95591529-b45ece80-0a47-11eb-82d0-b16fede7b0fa.gif)